src/services/users_service.py

```diff
@@ -18,12 +25,12 @@ async def get_user_by_id(self, uow: UnitOfWork, user_id: int, jwt_token: str | N
         Raises:
             Exception: If the user with the specified ID is not found.
         """
-        current_user = await self.get_current_user(jwt_token, uow)
+        current_user = await auth_service.get_current_user(jwt_token, uow)
         if not current_user:
             raise UserNotAuthenticatedError
 
         async with uow:
-            user = await uow.users.find_one(id=user_id)
+            user = await uow.users.find_one_or_none(id=user_id)
```

Comment:
try to use walrus operator.
for example:
if user := uow.users.find....:
     return user
raise Erorr